etc/.gitignore

```diff
@@ -1 +1,4 @@
 /flux
+/flux.service
+/rc1
+/rc3
```

Comment:
Thanks for fixing this, but as before it would be nice (not mandatory though) if the `.gitignore` changes unrelated to jobspec were under a separate commit. (just my .02)